crates/rome_js_analyze/src/utils/rename.rs

```diff
@@ -88,6 +88,36 @@ pub trait RenameSymbolExtensions {
         new_name: &str,
     ) -> bool;
 
+    /// Succesively try to rename a symbol using an iterator
+    /// as source for candidate names.
```

Comment:
What "Successively try" mean in this case? These two words seem to conflict with each other. Maybe we can reword it:

```suggestion
    /// It tries to rename a symbol using an iterator
    /// as source for candidate names. It returns a [bool], which is
    /// the result of the operation. `false` means that the renaming didn't occur. 
```

Reply:
I renamed the method and changed the comments.